src/pages/Help.vue

```diff
@@ -71,7 +71,7 @@
                 <p>That's because you don't legally own the game. The manager only supports legal copies.</p>
                 <hr/>
                 <h2 class='title is-5'>A text window appears and closes immediately.</h2>
-                <p>Try running the preloader fix on the Settings screen.</p>
+                <p>Try running "Validate Steam game installation" on the Settings screen.</p>
```

Comment:
Not necessary for this PR as the functionality is the same - Might be nice to make it an action that can actually run the setting? If it's high effort then not worth the hassle.

Reply:
Didn't include this now. Triggering the process would be simple enough, but I feel we'd need to describe what the actions does in more detail to avoid unexpected data loss, i.e. duplicate what the settings page currently says about the feature. Also we can't currently trigger the modal used by the settings page, as the help page isn't a subpage of the manager page.